### PR TITLE
Add livy REST API support for class GetLogResponse, GetStateResponse, GetStatementsReponse, PostStatements and PostCancelResponse

### PR DESCRIPTION
**Pull request by t-rufang**

### CODE REVIEW

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java [hunk 1]
```diff
@@ -0,0 +1,41 @@
...
+
+import com.microsoft.azure.hdinsight.sdk.rest.livy.interactive.Statement;
+
+import java.util.List;
+
+/**
+ * Returns all the statements in a session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ */
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java]: Needs more explaining here

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java [hunk 2]
```diff
@@ -0,0 +1,41 @@
...
+
+public class GetStatementsResponse {
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java]: Needs to implement IConvertible

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/PostStatements.java [hunk 3]
```diff
@@ -0,0 +1,41 @@
...
+
+package com.microsoft.azure.hdinsight.sdk.rest.livy.interactive.api.session;
+
+/**
+ * Runs a statement in a session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ */
+
+public class PostStatements {
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/PostStatements.java]: As above.

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java (hunk 2 above)
Comment (t-rufang) [reply Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java]: just to clarify, do we need to implement Iconvertible for all the classes of livy REST API?

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLog.java [hunk 4]
```diff
@@ -0,0 +1,56 @@
...
+
+import com.microsoft.azure.hdinsight.sdk.rest.IConvertible;
+
+/**
+ * Get the log lines from this session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ *
+ * For the following URI:
+ *   http://<livy base>/sessions
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLog.java]: `http://<livy base>/sessions/<sessionId>/log`

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLogResponse.java [hunk 5]
```diff
@@ -0,0 +1,65 @@
...
+
+import java.util.List;
+
+/**
+ * The response body after getting the log lines from this session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ *
+ * For the following URI:
+ *   http://<livy base>/sessions
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLogResponse.java]: `http://<livy base>/sessions/<sessionId>/log`

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLog.java [hunk 6]
```diff
@@ -0,0 +1,56 @@
+/*
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetLog.java]: Sorry, my mistake, no need for this class. The log GET parameters is not in request body.

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStateResponse.java [hunk 7]
```diff
@@ -0,0 +1,51 @@
...
+
+package com.microsoft.azure.hdinsight.sdk.rest.livy.interactive.api.session;
+
+/**
+ * Returns the state of session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ *
+ * For the following URI:
+ *   http://<livy base>/sessions
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStateResponse.java]: fix the URI please.

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java [hunk 8]
```diff
@@ -0,0 +1,51 @@
...
+
+import java.util.List;
+
+/**
+ * Returns all the statements in a session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ *
+ * For the following URI:
+ *   http://<livy base>/sessions
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/GetStatementsResponse.java]: Fix the uri please

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/PostStatements.java [hunk 9]
```diff
@@ -0,0 +1,52 @@
...
+
+import com.microsoft.azure.hdinsight.sdk.rest.IConvertible;
+
+/**
+ * Runs a statement in a session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ *
+ * For the following URI:
+ *   http://<livy base>/sessions
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/PostStatements.java]: fix the uri please

File: Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/statement/PostCancelResponse.java [hunk 10]
```diff
@@ -0,0 +1,46 @@
...
+
+package com.microsoft.azure.hdinsight.sdk.rest.livy.interactive.api.session.statement;
+
+/**
+ * Cancel the specified statement in this session.
+ *
+ * Based on Apache Livy, v0.4.0-incubating, refer to http://livy.incubator.apache.org./docs/0.4.0-incubating/rest-api.html
+ *
+ * For the following URI:
+ *   http://<livy base>/sessions
```
Comment (wezhang) [inline Utils/hdinsight-node-common/src/com/microsoft/azure/hdinsight/sdk/rest/livy/interactive/api/session/statement/PostCancelResponse.java]: fix the uri please

**Pull request merged**